### PR TITLE
Qe merge

### PR DESCRIPTION
**Pull request by TimSimpsonR**
This allows the tests to work outside of a VM.

### CODE REVIEW

File: scripts/conf/test.conf [hunk 1]
```diff
@@ -1,20 +1,25 @@
 {
+    "dbaas_url":"http://localhost:8779/v0.1",
```
Comment (rnirmal) [inline scripts/conf/test.conf:2]: isn't that 1.0

File: scripts/conf/test.conf [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+    "version_url":"http://localhost:8779",
+    "nova_auth_url":"http://localhost:35357/v2.0",
```
Comment (rnirmal) [inline scripts/conf/test.conf:4]: Do we need two of those... it's one auth url... we use the same auth service

Comment (TimSimpsonR) [inline scripts/conf/test.conf:4]: Probably not, but the code thinks it does now. I plan on taking some of this duplicate test.conf stuff out of the code soon, but not on this PR.

File: scripts/conf/test.conf (hunk 1 above)
Comment (TimSimpsonR) [inline scripts/conf/test.conf:2]: I was bewildered by that too. We can look into changing it later, but I want to keep it the same for this PR. I mostly moved all this stuff up so when QE makes their own config files these first properties (before the blank line), which contain endpoints and will be different, will stand out.

File: tests/integration/tests/util/users.py [hunk 3]
```diff
@@ -71,21 +74,30 @@ def __init__(self, user_list):
                                requirements=reqs)
             self.users.append(user)
 
-    def find_all_users_who_satisfy(self, requirements):
+    def find_all_users_who_satisfy(self, requirements, black_list = None):
```
Comment (ed-) [inline tests/integration/tests/util/users.py]: My bologna has a first name, and it's P E P 8

**Comment by cp16net** [comment]
+1

**Comment by ed-** [comment]
The requirements read much easier now.

+1

**Comment by sacharya** [comment]
+1

File: tests/integration/setup.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+
+
+# Copyright 2010 United States Government as represented by the
```
Comment (rnirmal) [inline tests/integration/setup.py]: I'm ok with copying copyrights. :)

File: tests/integration/tests/api/instances.py [hunk 5]
```diff
@@ -245,12 +241,13 @@ class CreateInstance(unittest.TestCase):
     """
 
     def test_instance_size_too_big(self):
-        if test_config.values['reddwarf_can_have_volume']:
-            raise SkipTest("Wrong assertion because we have no volume limit")
+        if 'reddwarf_max_accepted_volume_size' in test_config.values:
```
Comment (rnirmal) [inline tests/integration/tests/api/instances.py:244]: I'm sure this is old code.. but this should be something in the quotas, more max size. We don't have to do it here, just pointing out what we might want in the future

File: tests/integration/setup.py (hunk 4 above)
Comment (ed-) [inline tests/integration/setup.py]: :police_car:

**Comment by rnirmal** [comment]
+1

File: tests/integration/setup.py (hunk 4 above)
Comment (TimSimpsonR) [inline tests/integration/setup.py]: Oh man... if NASA ever sues us how will I say I came up with this in a clean room?

**Pull request merged**